.github/workflows/auto_community_labeler.yml

```
@@ -0,0 +1,28 @@
+
+name: Auto Community Labeler
+
+on:
+  pull_request:
+  issues:
+
+jobs:
+  auto_PR_labeler:
+    name: Auto PR Community Labeler
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' && github.event.pull_request.author_association != 'MEMBER' && github.event.pull_request.author_association != 'OWNER'
+
+    steps:
+      - name: Add Contributer Label
+        uses: actions-ecosystem/action-add-labels@v1
+        with:
+          labels: community contribution
+
+  auto_Issue_labeler:
+    name: Auto Issue Community Labeler
+    runs-on: ubuntu-latest
+    if: github.event_name == 'issues' && github.event.issue.author_association != 'MEMBER' && github.event.issue.author_association != 'OWNER'
+    steps:
+      - name: Add Contributer Label
```

There's a typo `Contributer` -> `Contributor`
```suggestion
      - name: Add Contributor Label
```